packages/tools/src/utilities/cine/events.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * CINE Tool Events
+ */
+enum Events {
+  CLIP_STOPPED = 'CORNERSTONE_CINE_TOOL_STOPPED',
```

Comment:
Is a clip started event useful? I can imagine some UI updating based on this? I.e. the user clicks a hotkey to start CINE, and then some UI updates to show its playing?